src/prompts/prompt.js

```diff
@@ -58,6 +59,13 @@ function prompt (options, done) {
   function focusDialog () {
     dom.input.focus();
   }
+  
+  function rootClick (e) {
+    var str = e.target.classList.value;
```

Comment:
Please rename `str` to something meaningful. Comment above the function would also be appreciated.